<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitDataViewPanel</class>
 <widget class="QWidget" name="OrbitDataViewPanel">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FilterLineEdit">
       <property name="accessibleName">
        <string>Filter</string>
       </property>
       <property name="placeholderText">
        <string>Filter</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="toolTip">
        <string>Refresh list</string>
       </property>
       <property name="accessibleName">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/orbiticons.qrc">
         <normaloff>:/actions/refresh</normaloff>:/actions/refresh</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="OrbitTreeView" name="treeView">
     <property name="accessibleName">
      <string>DataView</string>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>0</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrbitTreeView</class>
   <extends>QTreeView</extends>
   <header>OrbitQt/orbittreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
